seaworthy/containers/provided.py

```diff
@@ -191,3 +209,39 @@ def broker_url(self):
         """ Returns a "broker URL" for use with Celery. """
         return 'amqp://{}:{}@{}/{}'.format(
             self.user, self.password, self.name, self.vhost)
+
+    def _setup_management(self):
+        """
+        Enable the management API plugin and install curl.
+        """
+        if self._management_available:
+            return
+        self.inner().exec_run(['apk', 'add', '--update', 'curl'])
```

Comment:
You probably want `--no-cache` instead of `--update`

Reply:
Oh and there is BusyBox `wget`?

Reply:
Busybox `wget` is very limited. Specifically, it doesn't let us set the HTTP method and thus we can't make `PUT` requests.